src/loc/LocCreationSteps.tsx

```diff
@@ -67,6 +67,10 @@ export default function LocCreationSteps(props: Props) {
             })();
         }
     }, [
+        client,
+        legalOfficer,
+        locToCreate,
+        refreshLocs,
```

Comment:
I know the actual call to `refreshLocs()` comes from a previous PR, but it's async and should maybe `await`ed ?

Reply:
Indeed.